fieldservice_geoengine/models/fsm_location.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018 Open Source Integrators
+# Copyright (C) 2023 - TODAY Pytech SRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class FSMLocation(models.Model):
+    _inherit = "fsm.location"
+
+    # Geometry Field
+    shape = fields.GeoPoint("Coordinate", compute="_compute_shape", store=True)
+    stage_name = fields.Char(related="stage_id.name", string="Stage Name")
+    custom_color = fields.Char(related="stage_id.custom_color", string="Stage Color")
+
+    @api.model
```

Comment:
2024-05-12 03:17:01,463 21891 WARNING odoo16demo odoo.api.create: The model odoo.addons.fieldservice_geoengine.models.fsm_location is not overriding the create method in batch